service_crategen/src/main.rs

```diff
@@ -307,6 +308,17 @@ pub use custom::*;
         }
 
         {
+            let src_dir = crate_dir.join("src");
+            let gen_file_path = src_dir.join("generated.rs");
+
+            let _ = rustfmt::run(rustfmt::Input::File(gen_file_path), &rustfmt::config::Config {
```

Comment:
Any way we can have this output timing information?  Be nice to have our build instrumented to reduce build times.